src/main/java/frc/robot/subsystems/Conveyor.java

```diff
@@ -139,6 +185,96 @@ public void stop() {
         back.stopMotor();
     }
 
+    @Override
+    public void periodic() {
```

Comment:
there should be some logic that makes sure the motors are turning before updating states (except for none) to prevent updates from geting bumped and there also needs to be logic to account for running the conveyer backwards if it is being manualy controlled